modules/administration/pages/tshoot-hostname-rename.adoc

```diff
@@ -49,25 +49,28 @@ The only mandatory parameter for the script is the newly configured IP address o
     If the server is not using the new hostname, the script fails.
     Be aware that this script refreshes the pillar data for all Salt minion: the time it takes to run depends on the number of registered Salt systems.
 . Re-configure your clients to make your environment aware of the new hostname and IP address.
-+
-In the Salt minion configuration file [path]``/etc/salt/minion``, you must make sure to specify the name of the new Salt master ({productname} Server):
+    In the Salt minion configuration file [path]``/etc/salt/minion``, you must make sure to specify the name of the new Salt master ({productname} Server):
 +
 ----
 master: <new_hostname>
 ----
-
-Once this is changed, restart the [literal]``salt-minion` process:
-
++
+. Once this is changed, restart the [literal]``salt-minion` process:
++
 ----
 systemctl restart salt-minion
 ----
++
+. In order to fully propagate the hostname to the minion configuration (in the repositories URL for instance) apply the high state.
```

Comment:
```suggestion

. To fully propagate the hostname to the minion configuration apply the high state.
  Applying the high state will update the hostname in the repository URLs.
```
Try to avoid parens `;)`